### ISSUE TITLE
[Bug] User friendly choice list does not handle long lists with >400 options

### ISSUE THREAD

**Issue by atbagga** [Azure/azure-devops-cli-extension#571]
Function **prompt_user_friendly_choice_list** fails if the list is long.

**Sample code to repro:** 

```
mylist = []
for i in range(1, 600):
    mylist.append("Hello world") 
choice = prompt_user_friendly_choice_list("Choose an option?", mylist)
print('choice is {}'.format(choice))
```

Error: 
**On selecting a choice there is a fatal error.**
![image](https://user-images.githubusercontent.com/11888714/57057798-1a7aea00-6cc9-11e9-839a-c0402b5e6344.png)

**Comment by atbagga** [comment]
This should be fixed with the above PR. 
But to have a better experience I have added pagination support to the list..

![PromptlistPagination](https://user-images.githubusercontent.com/11888714/58624232-641d1b80-82ed-11e9-8bc1-5692af8d78bf.gif)


@geverghe Please take a look and suggest any improvements or corrections you would like to see.

**Comment by atbagga** [comment]
Bug is fixed and will be available in next release 0.10.0.
ETA of release: Early next week.
